resources/metabase-plugin.yaml

```diff
@@ -19,10 +19,24 @@ driver:
       display-name: Establish a read-only connection
       default: false
       type: boolean
+    - name: motherduck_token
+      display-name: Motherduck Token
+      type: secret
+      secret-kind: password
+      required: false
+      visible-if:
+        advanced-options: true
     - name: old_implicit_casting
       display-name: Use old_implicit_casting
       default: false
       type: boolean
+    - advanced-options-start
+    - merge:
+        - additional-options
```

Comment:
I'm not too familiar with the spec for this yaml but does this need a `type` field?

Reply:
This is some metabase preset i see applied to other drivers as well. it gets interpreted as a known set of configs so types are already known.